### DISCUSSION TITLE
Schema does not contain a key named 'last-directory'

### DISCUSSION THREAD

**Discussion by r-vorlicek** [phase1geo/Minder#275] [Q&A]
Hello.
When I click on "Save file as" in Minder window I get this error:
(com.github.phase1geo.minder:33166): GLib-GIO-ERROR **: 08:03:56.070: Settings schema 'com.github.phase1geo.minder' does not contain a key named 'last-directory'.

Removing all from minder directories in ~/.local and ~/.config doesn't help me.

Could you navigate me how fix it? or it's bug

version: ae7ee16132
os: Fedora 33

Thank you.

Robert

PS: PF2021

**Comment by r-vorlicek** [comment]
I'm sorry for disturbing you.
"sudo app install" resolves it.

**Reply by phase1geo** [reply]
I was just about to say that.
